### PR TITLE
Update BeginningTraining.md 2023

### PR DESCRIPTION
**Pull request by TimB-87**
Moved some things around in the first three weeks, added "Projects" section for the weeks that have projects, changed meeting to week, added 4 extra weeks.

### CODE REVIEW

File: Training Plan/BeginningTraining.md [hunk 1]
```diff
@@ -58,10 +59,20 @@
...
- - [ ] Functions
-  - Way of encapsulating code that is re-used
+ ### Projects
+- [ ] Create console programs / Review
+  - Write a 2 function calculator
+  - Write a 4 function calculator (Advanced)
+## Fourth Week:
+- Git, git etiquette, and how to run robot code
+Workflow and bug fixing (TEACH BRANCHING IN PREP FOR FUNCTIONS)
+((( (- [ ] Averaging Robot exercise
```
Comment (msoucy) [inline Training Plan/BeginningTraining.md:69]: Punctuation attack!

File: Training Plan/BeginningTraining.md [hunk 2]
```diff
@@ -124,9 +144,23 @@
...
+ ### Projects
+  - Basic subsystem with mock motors and sensors
+## Tenth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
 
 ## Filler Material
-For when the going gets tough and there isn't enough material to fill the time
+For when the going gets tough and there isn't enough material to fill the tim
```
Comment (msoucy) [inline Training Plan/BeginningTraining.md:163]: Please don't fill the tim

File: Training Plan/BeginningTraining.md [hunk 3]
```diff
@@ -124,9 +144,23 @@
...
+ ### Projects
+  - Basic subsystem with mock motors and sensors
+## Ninth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
+## Tenth Week:
+ - Interest splits: Auto, drivetrain, arm, LEDs
+ ### Projects
+  - Basic subsystem with mock motors and sensors
```
Comment (bot190) [inline Training Plan/BeginningTraining.md:160]: More detail here? Matt and I can help fill in details if you can add more direction.

**Review by bot190 — CHANGES_REQUESTED** [review]
It looks like you've stretched the training schedule out but not added more content, how do we plan to spend the extra time?

**Pull request closed without merging**